proxy/server.js

```diff
@@ -13,8 +13,8 @@ const app = express();
 process.env.UV_THREADPOOL_SIZE = 20;
 
 let counter = 0;
-// const BASEURL = "https://nodes.jup.io"; // prod url
-const BASEURL = "https://test.jup.io"; // test url
+const BASEURL = "https://nodes.jup.io"; // prod url
```

Comment:
i thought the proxy was to stay on the test endpoint? would at least like to discuss this

Reply:
i had been trying to keep it as the prod URL each time I merged to master but that's probably no longer a good idea now that tests have been re-tailored to the testnet URL, so I'll keep it as the test URL moving forward